english/mysterious-pool/giant-bubble/into-the-bubble/breathe-again.md

```diff
@@ -1,3 +1,8 @@
 You swim into the bubble, and gasping, you take a big breath in. You look
 around in wonderment at the soothing blue aura of the bubble. Just as you
-start to relax, thinking you may be alright after all, you realize...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.